src/partitionmanager.cpp

```diff
@@ -59,7 +59,7 @@ PartitionManagerPrivate::PartitionManagerPrivate()
 
     sharedInstance = this;
     m_udisksMonitor.reset(new UDisks2::Monitor(this));
-    connect(m_udisksMonitor.data(), &UDisks2::Monitor::status, this, &PartitionManagerPrivate::status);
```

Comment:
Elsewhere no one subscribes to the `PartitionManagerPrivate::status` signal? the logic of interaction will not break?

Reply:
Nobody does

Reply:
Doesn't anyone  this signal use  in the ui?

Reply:
Noone does. It hung

Reply:
Ok. Returned to quick-fix version